geode-core/src/test/java/org/apache/geode/management/internal/cli/CommandManagerJUnitTest.java

```diff
@@ -45,46 +37,6 @@
 public class CommandManagerJUnitTest {
 
   private static final String COMMAND1_NAME = "command1";
```

Comment:
I love seeing the purge, but should `COMMAND1_NAME` not also be on that chop block?